doc/source/starting.rst

```diff
@@ -720,6 +720,342 @@ For example:
+   >>> montepy.Nuclide("Co")
+   Nuclide('Co-0')
+
+
+.. note::
+
+   The new SZAID and Name syntax for nuclides introduced with MCNP 6.3.1 is not currently supported by MontePy.
+   This support likely will be added soon but probably not prior to MCNP 6.3.1 being available on RSICC. 
+
+
+Working with Material Components
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Iterating over Material Components
+""""""""""""""""""""""""""""""""""
+
+Materials are list-like iterables of tuples.
+
+.. testcode:: 
+
+    mat = problem.materials[1]
+
+    for comp in mat:
+        print(comp)
+
+This shows:
+
+.. testoutput::
+
+    (Nuclide('U-235.80c'), 5.0)
+    (Nuclide('U-238.80c'), 95.0)
+
+If you need just the nuclide, or just the fractions these are accessible by:
+:func:`~montepy.data_inputs.material.Material.nuclides`, and 
+:func:`~montepy.data_inputs.material.Material.values`.
+
+.. testcode::
+
+    for nuclide in mat.nuclides:
+        print(repr(nuclide))
+    for fraction in mat.values:
+        print(fraction)
+
+shows:
+
+.. testoutput::
+
+    Nuclide('U-235.80c')
+    Nuclide('U-238.80c')
+    5.0
+    95.0
+
+Updating Components of Materials
+""""""""""""""""""""""""""""""""
+
+Materials are also list-like in that they are settable by index.
+The material must always be set to a tuple of a nuclide and a fraction.
+
+For instance:
+
+.. testcode::
+
+    nuclide = mat[0][0]
+    mat[0] = (nuclide, 4.0)
+
+Generally this is pretty clunky, and so 
+:func:`~montepy.data_inputs.material.Material.nuclides` and 
+:func:`~montepy.data_inputs.material.Material.values` are also settable.
+To undo the previous changes:
+
+.. testcode::
+
+    mat.values[0] = 5.0
+    print(mat[0])
+
+This outputs: 
+
+.. testoutput::
+
+    (Nuclide('U-235.80c'), 5.0)
+
+Adding Components to a Material
+"""""""""""""""""""""""""""""""
+
+To add components to a material use either
+:func:`~montepy.data_inputs.material.Material.add_nuclide`, or
+:func:`~montepy.data_inputs.material.Material.append`.
+:func:`~montepy.data_inputs.material.Material.add_nuclide` is generally the easier method to use.
+It accepts a nuclide or the name of a nuclide, and its fraction.
+
+.. note::
+
+    When adding a new component it is not possible to change whether the fraction is in atom fraction 
+    or mass fraction.
+    This is settable through :func:`~montepy.data_inputs.material.Material.is_atom_fraction`.
+
+.. testcode::
+
+    mat.add_nuclide("B-10.80c", 1e-6)
+    for comp in mat:
+        print(comp)
+
+.. testoutput::
+
+    (Nuclide('U-235.80c'), 5.0)
+    (Nuclide('U-238.80c'), 95.0)
+    (Nuclide('B-10.80c'), 1e-06)
+
+
+Libraries
+^^^^^^^^^
+
+MCNP nuclear data comes pre-packaged in multiple different libraries that come from different nuclear data sources
+(e.g., ENDF/B-VIII.0),
+at different temperatures, 
+and for different data needs, e.g., neutron data vs. photo-atomic data.
+For more details see `LA-UR-17-20709 <https://www.osti.gov/biblio/1342828>`_, or 
```

Comment:
OSTI link or [LANL permalink](https://permalink.lanl.gov/object/tr?what=info:lanl-repo/lareport/LA-UR-17-20709)?

Reply:
I never trust a LANL permalink. They said they were permalinks about two years ago before they all changed. So for here I think OSTI is better. I am fine with the manual being to LANL because it's only one place to fix in the `conf.py`